lib/dgram.js

```diff
@@ -294,8 +301,7 @@ Socket.prototype.send = function(buffer,
                                  callback) {
   var self = this;
 
-  // same as arguments.length === 5 || arguments.length === 6
-  if (address) {
+  if (address || port && typeof port !== 'function') {
```

Comment:
sorry, a very minor nit but I'd prefer brackets around the `&&` section to be explicit although I don't know if we've set precedent elsewhere in the code